Makefile

```diff
@@ -1,5 +1,5 @@
 NAME = skippy/oauth2_proxy
-VERSION = 2.0.1
+VERSION = 2.1
```

Comment:
Seems as if version of skippy/oauth2_proxy is synchronized with oauth2_proxy version. Maybe "2.2" here?!